VultisigApp/VultisigApp/Services/BlockChainService.swift

```diff
@@ -19,8 +19,13 @@ final class BlockChainService {
         return value * 2 + value / 2 // x2.5 fee
     }
     
-    static func normalizeEVMFee(_ value: BigInt) -> BigInt {
-        let normalized = value + value / 2 // x1.5 fee
+    static func normalizeEVMFee(_ value: BigInt, chain: Chain? = nil) -> BigInt {
+        var multiplier: BigInt = 3
+        if chain == .base {
+            multiplier = 5 // x2.5 fee for Base chain
+        }
+        
+        let normalized = value + (value / 2) * (multiplier / 3)
```

Comment:
try to think about this math again , you are not getting 2.5x 
value + (value/2) * (5/3) , you are getting around 1.8x